http-add-on/templates/interceptor/deployment.yaml

```diff
@@ -66,6 +66,8 @@ spec:
           value: "{{ .Values.interceptor.xds.keepalive.timeout }}"
         - name: KEDIFY_XDS_KEEPALIVE_ENFORCEMENT_POLICY_MIN_TIME
           value: "{{ .Values.interceptor.xds.keepalive.enforcementPolicyMinTime }}"
+        - name: KEDIFY_SETUP_WILDCARD_DEFAULT_ROUTE
```

Comment:
Not totally sure about the naming, it's too generel, especially the `SETUP` part, but I also don't have an idea what would be better 😄

Reply:
yeah, I didn't love it either, what about `KEDIFY_ENVOY_ENABLE_CATCH_ALL_ROUTE`?